src/core/server/status/status_service.ts

```diff
@@ -192,4 +212,40 @@ export class StatusService implements CoreService<InternalStatusServiceSetup> {
       shareReplay(1)
     );
   }
+
+  private setupAnalyticsContextAndEvents(analytics: AnalyticsServiceSetup) {
+    // Set an initial "initializing" status, so we can attach it to early events.
+    const context$ = new BehaviorSubject<StatusAnalyticsPayload>({
```

Comment:
I was worried that this new Subject might not be completed when `overall$` is.
On the same lines I was wondering if we could get away without creating a new Subject, and using something like `startWith` instead.

<img width="985" alt="image" src="https://user-images.githubusercontent.com/25349407/166970752-9ea1240e-5da4-4dfa-8a54-245efc8bc2c6.png">

Tried `startWith` and `concat`, but I can't get the same behavior. At the same time, I confirmed that `context$` is properly "completed" whenever `overall$` is.

Props to your implementation 👏🏼